contracts/utils/Create2.sol

```diff
@@ -31,16 +31,14 @@ library Create2 {
         uint256 amount,
         bytes32 salt,
         bytes memory bytecode
-    ) internal returns (address) {
-        address addr;
+    ) internal returns (address addr) {
```

Comment:
It that change optimizing anything? If not, I would revert it and focus on the `computeAddress` function